docs/content/preview/explore/query-1-performance/explain-analyze.md

```diff
@@ -2,455 +2,274 @@
-    Index Cond: (account_id = 8060)
-        Filter: ((NOT is_paid) AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone))
-(4 rows)
-
-Time: 46.658 ms
-Previous run time: 147 seconds
-```
-
-### Optimize `SELECT` by changing table sorting
-
-The following query retrieves data from an account table where some indexes are already defined.
-
-Table definition:
-
-```sql
-yugabyte=# \d accounts
-```
-
-```output
-                                    Table "public.accounts"
-Column        | Type                        | Collation | Nullable | Default
---------------+-----------------------------+-----------+----------+--------
-id            | bigint                      | For       | not null | nextval('accounts_id_seq'::regclass)
-company_name  | character varying(125)      |           | not null |
-status        | character varying(25)       |           |          |
-first_name    | character varying(55)       |           |          |
-last_name     | character varying(55)       |           |          |
-phone         | character varying(25)       |           |          |
-created_at    | timestamp without time zone |           | not null |
-updated_at    | timestamp without time zone |           | not null |
-product_id    | integer                     |           |          |
-business_type | character varying(55)       |           |          |
-sales_rep     | character varying           |           |          |
-
-Indexes:
-    "accounts_pkey" PRIMARY KEY, lsm (id HASH)
-    "index_on_company_name" lsm (company_name HASH)
-```
-
-`EXPLAIN` output for the query is as follows:
+EXPLAIN (ANALYZE, DIST, COSTS OFF) SELECT * FROM kvstore WHERE value LIKE 'ca%' ORDER BY VALUE;
+```
+
+You should see an output similar to,
+
+```yaml{linenos=inline, .nocopy}
+                                 QUERY PLAN
+-----------------------------------------------------------------------------
+ Sort (actual time=4.007..4.032 rows=41 loops=1)
+   Sort Key: value
+   Sort Method: quicksort  Memory: 28kB
+   ->  Seq Scan on kvstore (actual time=3.924..3.957 rows=41 loops=1)
+         Storage Filter: ((value)::text ~~ 'ca%'::text)
+         Storage Table Read Requests: 1
+         Storage Table Read Execution Time: 3.846 ms
+         Storage Table Rows Scanned: 10000
+ Planning Time: 1.119 ms
+ Execution Time: 4.101 ms
+ Storage Read Requests: 1
+ Storage Read Execution Time: 3.846 ms
+ Storage Rows Scanned: 10000
+ Catalog Read Requests: 1
+ Catalog Read Execution Time: 0.877 ms
+ Storage Execution Time: 4.723 ms
+```
+
+Just 41 rows were returned (`rows=41`) but a sequential scan (`Seq Scan on kvstore`) was performed on all the rows (`Storage Rows Scanned: 10000`). Also the data had to be sorted (via `Sort Method: quicksort`) on value (via `Sort Key: value`). We can improve this execution by modifying the above created index into a range index, so that we can avoid the sorting operation and reduce the number of rows scanned.
```

Comment:
```suggestion
Only 41 rows were returned (`rows=41`) but a sequential scan (`Seq Scan on kvstore`) was performed on all the rows (`Storage Rows Scanned: 10000`). Also, the data had to be sorted (via `Sort Method: quicksort`) on value (via `Sort Key: value`). You can improve this execution by modifying the above created index into a range index, so that you can avoid the sorting operation, and reduce the number of rows scanned.
```